cg/store/api/find_business_data.py

```diff
@@ -314,12 +314,18 @@ def get_sample_by_customer_and_name(
             name=sample_name,
         ).first()
 
-    def get_number_of_reads_for_sample_from_metrics(self, sample_internal_id: str) -> int:
+    def get_number_of_reads_for_sample_from_metrics(
+        self, sample_internal_id: str, q30_threshold: int
+    ) -> int:
         """Get number of reads for sample from sample lane sequencing metrics."""
```

Comment:
```suggestion
        """Get number of reads that pass bases percent above q30 threshold for sample from sample lane sequencing metrics."""
```

Reply:
I think this doc string is a DRY violation (the name contains the exact same info). Should we remove doc strings like these instead?

Reply:
I think the info that there is a threshold and exactly what table the metrics are taken from does make it useful and doesn't violate DrY

Reply:
I guess, there is also the risk of leaking implementation details. Does the caller really need to know which table is used? The caller is only interested in the result and might not care about the actual implementation

Reply:
Updated the doc string